Formula/q/quick-lint-js.rb

```diff
@@ -19,7 +19,6 @@ class QuickLintJs < Formula
   depends_on "cmake" => :build
   depends_on "googletest" => :build
   depends_on "pkg-config" => :build
-  depends_on "boost"
```

Comment:
quick-lint-js no longer depends on Boost.

Reply:
@strager can you also link the relevant PR or commit as well? Thanks!

Reply:
https://github.com/quick-lint/quick-lint-js/commit/ac1e2fb9f5178345e817a8537d99e7e883f660cd